driver/kubernetes/factory_test.go

```diff
@@ -192,6 +210,30 @@ func TestFactory_processDriverOpts(t *testing.T) {
 		},
 	)
 
+	t.Run(
+		"InvalidCustomAnnotation", func(t *testing.T) {
+			cfg.DriverOpts = map[string]string{
+				"annotations": "key,value",
+			}
+
+			_, _, _, err := f.processDriverOpts(cfg.Name, "test", cfg)
+
```

Comment:
nit: remove extra lines

Reply:
I applied this comment on the other identical surrounding tests as well. I also noticed an extra `ns := "test"` at the beginning that was not necessary since it was rewritten in the next call.